src/components/MenuButton/MenuButton.md

```diff
@@ -0,0 +1,76 @@
+### Notes for use
+
+Context menu should be used as a menu button where you want to provide options that are hidden behind a click.
+
+### Examples
+
+MenuButton with default Icon, and all applicable children properties
+
+```js { "props": { "data-description": "with deafult icon and children properties" } }
```

Comment:
"default" is misspelled.

how about "with default icon and all menu item types"